crates/subspace-node/src/secondary_chain/cli.rs

```diff
@@ -38,7 +38,7 @@ pub enum Subcommand {
 
     /// Sub-commands concerned with benchmarking.
     #[clap(subcommand)]
-    Benchmark(frame_benchmarking_cli::BenchmarkCmd),
```

Comment:
What is the reason for it to be boxed?

Reply:
Something must have changed in Substrate and clippy started to complain about large difference in size between enum variants